console/src/App/AppContextConnect.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Pure from '@enact/ui/internal/Pure';
+import {AppContext} from './AppContextProvider'
+
+const AppStateConnect = (mapContextToProps) => (Wrapped) => {
```

Comment:
This looks like a nice reusable bit of code. Perhaps we need something generalized like this in UI. I just yesterday encountered a scenario where I needed to write basically this too. (Mine was less pretty.)

Reply:
Agreed, but we'll need to figure out a good way to make it extensible from the provider section. But yeah i think even just having a basic catch all to update things would be nice.